frontend/src/utils/QRGen.ts

```diff
@@ -0,0 +1,37 @@
+import crypto from 'crypto';
+import { KeygenMessage } from '../gen/vultisig/keygen/v1/keygen_message_pb';
+import SevenZip from '7z-wasm';
```

Comment:
This package is not being maintained anymore, can we find alternatives?

https://snyk.io/advisor/npm-package/7z-wasm

Reply:
I actually used alternative one in vultisigner but seems like this 7z-wasm is most compatible with ios and android compression. We had an uncompatible issue in vultisigner.
Please note that this package is still valuable and weekly downloads are more than 1k and last version released in less than a year so we can still use it.